shared/src/commonMain/kotlin/ui/navigation/WalletNavigation.kt

```diff
@@ -272,7 +272,7 @@ private fun WalletNavHost(
             val route: AuthenticationViewRoute = backStackEntry.toRoute()
 
             val vm = try {
-                val request = odcJsonSerializer
+                val request = rdcJsonSerializer
```

Comment:
@n0900 Otherwise, it can't parse an authorization request containing `transaction_data` (encoded in the wrong "old" way) ... this got me thinking, do we need to change that everywhere?!